Soomsil-USaint/Data/Client/RuSaint/GradeClient.swift

```diff
@@ -4,4 +4,253 @@
 //
 //  Created by 정지혁 on 1/9/25.
 //
+import CoreData
 
+import ComposableArchitecture
+import Rusaint
+
+struct GradeClient {
```

Comment:
```suggestion
@DependencyClient
struct GradeClient {
```

`@DependencyClient` 매크로를 추가하는게 아키텍쳐에 적합해보여요!

Reply:
넵! 반영해놓겠습니다👍